### PR TITLE
feat(/calendar,/originstory,/events,/about): Implement UI 

### PR DESCRIPTION
**Pull request by avvsrohith**
In this Pull Request.

- Implemented design and UI for /originStory, /about, /calendar, /events.
- Added required assets.
- Implemented modular structure by including widgets of same page in a folder

### CODE REVIEW

File: lib/pages/events_page.dart [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  void initState() {
+    super.initState();
+    _tabController = TabController(length: 5, vsync: this);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        key: _scaffoldKey,
+        appBar: AppBar(
```
Comment (Ram-20062003) [inline lib/pages/events_page.dart]: We are not having an app bar in the app. so remove this.
And see how the navicon was implemented in authpage

File: lib/pages/originstory_page.dart [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  void initState() {
+    super.initState();
+    _tabController = TabController(length: 6, vsync: this);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: _scaffoldKey,
+      appBar: AppBar(
```
Comment (Ram-20062003) [inline lib/pages/originstory_page.dart]: Same here

File: lib/utils/constants.dart [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class ClientCredentials {
```
Comment (Ram-20062003) [inline lib/utils/constants.dart]: Creditionals should be in config. dart.Pls, clear this.

File: lib/pages/calendar_page.dart [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: _scaffoldKey,
+      appBar: AppBar(
```
Comment (Ram-20062003) [inline lib/pages/calendar_page.dart]: Same remove app bar

File: lib/pages/about_page.dart [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    Cup(
+        imageasset: 'assets/images/Aaveg Glyph - White 1.png',
+        desc: 'This text is to describe the cup'),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: _scaffoldKey,
+      appBar: AppBar(
```
Comment (Ram-20062003) [inline lib/pages/about_page.dart]: Here also again

**Comment by SuhailAhmed2627** [comment]
Pull Rebase

File: lib/pages/about_page.dart [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+        drawer: Theme(
+            data: Theme.of(context).copyWith(
+              canvasColor: Colors.black.withOpacity(0.4),
+            ),
+            child: Drawer(
+              child: NavbarWdiget(),
+            )),
+        body: Container(
+          constraints: BoxConstraints.expand(),
+          decoration: BoxDecoration(
```
Comment (Ram-20062003) [inline lib/pages/about_page.dart:65]: Background image takes some margin here ig .Add fit parameter here as Boxfit.fill

File: lib/pages/about_page.dart [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+              image: DecorationImage(
```
Comment (Ram-20062003) [inline lib/pages/about_page.dart:66]: Add black overlay also.

File: lib/views/widgets/Events/event_card.dart [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+                        ),
+                      ),
+                      Expanded(
+                        flex: 35,
+                        child: Center(
+                          child: Text(
+                            widget.event.name,
+                            style: TextStyle(
+                                fontWeight: FontWeight.bold,
+                                fontSize: totalHeight * 0.024,
```
Comment (Ram-20062003) [inline lib/views/widgets/Events/event_card.dart]: make font bigger this is too small for smaller screen mobiles.

File: lib/views/widgets/Events/event_card.dart [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+                                fontFamily:
+                                    GoogleFonts.montserrat().fontFamily),
+                          ),
+                        ),
+                      ),
+                      Expanded(
+                        flex: 25,
+                        child: Center(
+                          child: Text(
+                            'click here for rulebook -->',
```
Comment (Ram-20062003) [inline lib/views/widgets/Events/event_card.dart]: Insert the poper straight line arrow as in figma. not dotted lines

File: lib/views/widgets/Events/event_card.dart [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+                      //   child: SizedBox(
+                      //     height: 25,
+                      //   ),
+                      // ),
+                      Expanded(
+                        flex: 25,
+                        child: Center(
+                          child: Text(
+                            widget.event.r1Date +
+                                '      ' +
```
Comment (Ram-20062003) [inline lib/views/widgets/Events/event_card.dart]: To the left of the dates there should be a calendar icon . it is not there.Check figma once

File: lib/views/widgets/Events/event_card.dart [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+                                        fontFamily: GoogleFonts.montserrat()
+                                            .fontFamily)),
+                              ),
+                              flex: 1),
+                          Expanded(
+                              child: Align(
+                                alignment: Alignment.centerLeft,
+                                child: Text(
+                                  'Know more',
+                                  style: TextStyle(
```
Comment (Ram-20062003) [inline lib/views/widgets/Events/event_card.dart]: it should be italics and should have arrow after it

File: lib/views/widgets/Events/event_card.dart [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+                                      fontFamily:
+                                          GoogleFonts.montserrat().fontFamily,
+                                      fontSize: totalHeight * 0.012),
```
Comment (Ram-20062003) [inline lib/views/widgets/Events/event_card.dart]: The font size here really makes the know more hidden.maintain a constant visible font-size

File: lib/views/widgets/About/about_text.dart [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+  _AboutTextWidgetState createState() => _AboutTextWidgetState();
+}
+
+class _AboutTextWidgetState extends State<AboutTextWidget> {
+  @override
+  Widget build(BuildContext context) {
+    double totalHeight = MediaQuery.of(context).size.height;
+    double totalWidth = MediaQuery.of(context).size.width;
+
+    return Container(
```
Comment (Ram-20062003) [inline lib/views/widgets/About/about_text.dart:17]: it is better to put a singlechildscrollview .because of lots of text will be coming later

**Review by Ram-20062003 — CHANGES_REQUESTED** [review]

**Comment by Ram-20062003** [comment]
Screenshot from 2022-06-10 14-56-03

**the text in calendar is really difficult to read** .Do look into it

**Review by Ram-20062003 — APPROVED** [review]

**Pull request merged**